<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="user-PC" failures="1" tests="1" name="Jewwy Test" time="6.511" errors="0" timestamp="2023-04-18T15:37:28 GST">
  <testcase classname="testcase.MyFirstTestFW" name="LoginTest" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 2">
      <![CDATA[java.lang.RuntimeException: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 2
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 2
at utilities.ReadXlData.getData(ReadXlData.java:42)
... 25 more
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- LoginTest -->
</testsuite> <!-- Jewwy Test -->
